src/styles/main.scss

```diff
@@ -1 +1,6 @@
-// put code here
+@import "../utils/variables";
+@import "../utils/reset";
+@import "../utils/keyframes";
+
+@import "./container.scss";
+@import "./stop-watch.scss";
```

Comment:
Once again, avoid using relative paths in import statements. Use absolute paths for a clear project structure and easier refactoring.